controllers/rcj_soccer_referee_supervisor/referee/supervisor.py

```diff
@@ -308,6 +308,9 @@ def _pack_packet(
         data.append(ball_translation[0])
         data.append(ball_translation[2])
 
+        # Add Notification if the goal is scored (True or False)
+        data.append(self.ball_reset_timer > 0)
```

Comment:
@Adman This is relatively nasty, this essentially mean the notification tells you whether we are in the time between when the goal gets scored and the new kickoff gets staged. I think calling it `is_goal` may be a bit dangerous in this regard (it is not set to true only in the "tick" when the goal gets scored).

Can you think of a name that's a bit more appropriate? I guess we should also explicitly describe this in the docs.

Thanks!

Reply:
Yeah, we should describe entire "packet" string in docs.

would something like `waiting_for_kickoff` or something similar work?

Reply:
@Adman that does indeed sound very nice -- thanks!

Reply:
Renamed.